src/app/routes/enrolment/view-requests/components/full-credential/full-credential.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-credential-json',
+  templateUrl: './full-credential.component.html',
+  styleUrls: ['./full-credential.component.scss'],
+})
+export class CredentialJsonComponent {
+  @Input() title: string;
+  @Input() credentialData: object;
+  @Input() expEnabled: boolean;
```

Comment:
In my opinion, it would be better to use `experimentalEnabled` rather than `expEnabled`. Best to avoid abbreviations in names, IMO. Actually, when I first read this, I thought it might have been "expiryEnabled".

Reply:
Resolving this as the flag has been removed.